telecine/src/main/java/com/jakewharton/telecine/RecordingSession.java

```diff
@@ -395,4 +405,24 @@ public void destroy() {
       stopRecording();
     }
   }
+
+  private static int getNotificationId() {
+    return (int) SystemClock.uptimeMillis();
+  }
+
+  public static class DeleteRecordingBroadcastReceiver extends BroadcastReceiver {
+
+    @Override public void onReceive(Context context, Intent intent) {
+      int rowsDeleted = context.getContentResolver().delete(intent.getData(), null, null);
```

Comment:
Shouldn't this be run an a background thread or the implementation already does that?